dts/bindings/timer/nxp,s32-emios-timer.yaml

```diff
@@ -0,0 +1,94 @@
+# Copyright 2023 NXP
```

Comment:
Why is this file added. I don't see a timer driver. If this module is intended for a timer driver then this binding should be included as part of that PR.

Reply:
No, this module is not intended for timer driver (although it could), i will push the update to move it out of here

Reply:
I created separate misc driver for eMIOS that does initializes global configuration for eMIOS